src/routes/engineimage/EngineImageList.js

```diff
@@ -31,10 +31,10 @@ function list({ loading, dataSource, deleteEngineImage }) {
       dataIndex: 'state',
       key: 'state',
       width: 100,
-      render: (text) => {
+      render: (text, record) => {
         return (
-          <div className={classnames({ [text.toLowerCase()]: true, capitalize: true })}>
-            {text.hyphenToHump()}
+          <div className={classnames({ [record.incompatible ? 'incompatible' : text.toLowerCase()]: true, capitalize: true })}>
```

Comment:
Does this class really take effect? I didn't see any style class `incompatible` or `deploying` have been defined, so I assume it did not work initially.

However, it's fine for now, since it did not have a right style before.

Reply:
cc @scures